src/test/java/bio/terra/service/filedata/google/firestore/FireStoreDirectoryDaoTest.java

```diff
@@ -271,10 +296,10 @@ public void bulkDirectoryOperationsFSObjectsTest() throws Exception {
   }
 
   @Test
-  public void testEnumerateFileRefEntries() throws InterruptedException {
+  void testEnumerateFileRefEntries() throws InterruptedException {
     List<FireStoreDirectoryEntry> noFiles =
         directoryDao.enumerateFileRefEntries(firestore, collectionId, 0, 10);
-    assertEquals(noFiles.size(), 0);
+    assertThat(noFiles.size(), equalTo(0));
```

Comment:
If you prefer, you could simplify your check thusly:
```suggestion
    assertThat(noFiles, empty());
```